packages/cli/src/integrate.ts

```diff
@@ -0,0 +1,71 @@
+import { installPackage, readFile, writeFile } from './utils';
+import fse from 'fs-extra';
+
+interface IntegrateOptions {
+  apiKey: string;
+  model: string;
+  pathPrefix: string;
+}
+
+const NEXTJS_PAGES_TEMPLATE_PATH = './src/templates/nextjs-[pages].tsx';
+
+function verifyPagesDirectory() {
+  return fse.existsSync('pages');
+}
+
+function checkForPagesFile(prefix: string, extension: string) {
+  return fse.existsSync(`./pages/${prefix}/[pages].${extension}`);
```

Comment:
couldn't `[pages]` be any other word though? I think you need to do a glob check for any file in `./pages/${prefix}` with the glob `[(.*)].${extension}` (or something like that)

Reply:
https://nextjs.org/docs/routing/dynamic-routes